src/components/resource/form.js

```diff
@@ -199,10 +214,12 @@ defineComponent('furet-ui-resource-form', {
             // new case
             this.readonly = false;
             this.uuid = this.manager.query.uuid;
+            this.getDefault();
           } else {
             // new case
             this.readonly = false;
             this.uuid = uuidv1();
+            this.getDefault();
```

Comment:
I'm wondering why parse_query is called twice while creating a new element ?!

looking for a better place...

Reply:
it's because created and manager watch call it. why both, are there really required ?